### PR TITLE
Add modelform-uses-exclude check

### PR DESCRIPTION
**Pull request by federicobond**
Refs #168. The message wording can be improved, but I wanted to get the ball rolling.

### CODE REVIEW

File: pylint_django/checkers/forms.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+from pylint_django.__pkginfo__ import BASE_ID
+from pylint_django.utils import node_is_subclass
+
+
+MESSAGES = {
+    'W%d04' % BASE_ID: ("Use explicit fields instead of exclude in ModelForm",
+                        'modelform-uses-exclude',
+                        "Replace exclude with fields attribute"),
+}
```
Comment (atodorov) [inline pylint_django/checkers/forms.py]: This dictionary is used only inside the checker class so move it there. No need to expose extra objects outside.

File: pylint_django/checkers/forms.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    """Django model checker."""
+    __implements__ = IAstroidChecker
+
+    name = 'django-form-checker'
+    msgs = MESSAGES
+
+    # NOTE: there's a bug in pylint's backwards compatability logic after changing
+    # visit method names from visit_class to visit_classdef, see
+    # https://bitbucket.org/logilab/pylint/issues/711/new-node-visit-methods-not-backwards
+    if PYLINT_VERSION < (1, 5):
```
Comment (atodorov) [inline pylint_django/checkers/forms.py]: setup.py requires 'pylint>=1.8.2' so this check is not necessary.

**Comment by federicobond** [comment]
Fixed.

**Comment by carlio** [comment]
Looks good to me after the changes - @atodorov want to double check? Otherwise I think it's good to merge.

File: pylint_django/checkers/forms.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+from pylint_django.__pkginfo__ import BASE_ID
+from pylint_django.utils import node_is_subclass
+
+
+def _get_child_meta(node):
+    for child in node.get_children():
+        if isinstance(child, ClassDef) and child.name == 'Meta':
+            return child
+    return None
```
Comment (atodorov) [inline pylint_django/checkers/forms.py]: You can remove this line b/c this is the default in Python. Indeed the unreleased pylint 2.0 has a check for this and will complain.

Comment (federicobond) [reply pylint_django/checkers/forms.py]: Yes, I added it because my installed version of Pylint marked it. I'll take it away.

Comment (carlio) [reply pylint_django/checkers/forms.py]: If pylint complains differently in different versions... well now you know why this PR is fixing the hacks I made :-)  see also https://github.com/PyCQA/pylint-plugin-utils/blob/master/pylint_plugin_utils/__init__.py#L109 ...

**Pull request merged**